samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Service.Country.g.cs

```diff
@@ -18,7 +18,7 @@ public ApiUiService IntialiseApiUiService()
     {
         ApiUiService rtnApiUiService = new();
 
-        rtnApiUiService.Url = "https://localhost:44310/api/Countries";
+        rtnApiUiService.Url = "https://localhost:44310/api/v1/Countries";
```

Comment:
Can we separate "https://localhost:44310/" into configs and generate relative path here? Maybe we even need a yaml property to define 'server url'

Reply:
Thanks @anna-naumova25  - yes I agree - currently it is on a separate ticket for now though - https://regusit.atlassian.net/browse/NOX-76 - we need to discuss as it it should be related to  Infrastructure.EndPoints.ApiServer yaml settings  - hence me creating these Application.Service classes to cover it for now if that is ok.

Reply:
Ok @anna-naumova25 on 2nd commit have moved BaseApiUrl in to appsettings config for now until we have a better solution on NOX-76 implemented thanks